src/Raven.Server/Commercial/LetsEncryptClient.cs

```diff
@@ -380,7 +380,7 @@ private async Task<HttpResponseMessage> SendAsyncInternal(HttpMethod method, Uri
 
         public async Task<(X509Certificate2 Cert, RSA PrivateKey)> GetCertificate(RSA existingKey = null, string acmeProfile = null, CancellationToken token = default(CancellationToken))
         {
-            var key = existingKey ?? new RSACryptoServiceProvider(4096);
+            var key = existingKey?.GetExportableRsaPrivateKey() ?? RSA.Create(4096);
```

Comment:
Can we default to `ECDsa` here? `RSA.Create` is _exepnsive_ especially on small machines / shared CPU

Reply:
It is a separate issue, this one is a 1:1 transition